src/subdomains/supporting/bank/fiat-output/fiat-output.entity.ts

```diff
@@ -0,0 +1,12 @@
+import { IEntity } from 'src/shared/models/entity';
+import { Entity, OneToOne, Column } from 'typeorm';
+import { BuyFiat } from '../../../core/sell-crypto/buy-fiat/buy-fiat.entity';
+
+@Entity()
+export class FiatOutput extends IEntity {
+  @OneToOne(() => BuyFiat, (buyFiat) => buyFiat.fiatOutput, { nullable: true })
+  buyFiat?: BuyFiat;
+
+  @Column({ length: 256, nullable: true })
+  reason: string;
```

Comment:
Maybe `type` is a better naming than `reason`, WDYT? Sorry, `reason` was my proposition 😇

Reply:
Reason should be required IMO (also in DTO)